src/PrestaShopBundle/Entity/Store.php

```diff
@@ -0,0 +1,325 @@
+<?php
+/**
+ * Copyright since 2007 PrestaShop SA and Contributors
+ * PrestaShop is an International Registered Trademark & Property of PrestaShop SA
+ *
+ * NOTICE OF LICENSE
+ *
+ * This source file is subject to the Open Software License (OSL 3.0)
+ * that is bundled with this package in the file LICENSE.md.
+ * It is also available through the world-wide-web at this URL:
+ * https://opensource.org/licenses/OSL-3.0
+ * If you did not receive a copy of the license and are unable to
+ * obtain it through the world-wide-web, please send an email
+ * to license@prestashop.com so we can send you a copy immediately.
+ *
+ * DISCLAIMER
+ *
+ * Do not edit or add to this file if you wish to upgrade PrestaShop to newer
+ * versions in the future. If you wish to customize PrestaShop for your
+ * needs please refer to https://devdocs.prestashop.com/ for more information.
+ *
+ * @author    PrestaShop SA and Contributors <contact@prestashop.com>
+ * @copyright Since 2007 PrestaShop SA and Contributors
+ * @license   https://opensource.org/licenses/OSL-3.0 Open Software License (OSL 3.0)
+ */
+
+declare(strict_types=1);
+
+namespace PrestaShopBundle\Entity;
+
+use Doctrine\Common\Collections\ArrayCollection;
+use Doctrine\ORM\Mapping as ORM;
+use PrestaShop\PrestaShop\Core\ConstraintValidator\Constraints as PrestaShopAssert;
+use PrestaShopBundle\Entity\Repository\StoreRepository;
+use Symfony\Component\Validator\Constraints as Assert;
+
+/**
```

Comment:
Hello as the PR does not contain the code that will make use of this class I cannot see if it's a good implementation or not.

I know you want to split the PR into small bits but if I don't have the context I cannot provide feedback about whether or not this code is valid.

However so far we have not used Doctrine entities for back-office listing pages. If you look at other back-office listing pages migrated to Symfony we do not use Doctrine entities. So this looks like a wrong path.

Reply:
Thought Doctrine migration was topical, my bad. I'll finish it and put it aside in the meantime.